openwisp_notifications/handlers.py

```diff
@@ -199,23 +196,14 @@ def send_email_notification(sender, instance, created, **kwargs):
         target_url = None
     if target_url:
         description += '\n\nFor more information see {0}.'.format(target_url)
```

Comment:
let's take advantage to flag `For more information see` as translatable, which doesn't seem to flagged as translatable now.

Please also add "Find out more" since I asked you to remove it from openwisp-utils.

Reply:
We are using `blocktrans` in our template which I think will take care of that.
Do we still need to do so here?

Reply:
This one? https://github.com/openwisp/openwisp-utils/blob/issues/259-email-template/openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html#L104
I didn't see that. It's totally useless, translations do not work that way, each string has to be flagged on its own.
Please study how translations work in Django and gettext here: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/, read carefully the parts in which it shows how to deal with placeholder of text which is inserted in the strings and how that has to be written  in order to work (f stings and str.format() do not work).